.gitignore

```diff
@@ -8,3 +8,4 @@ local.properties
 package-lock.json
 .DS_Store
 node_modules/
+apks/
```

Comment:
should we also add `app/build` ?

Reply:
`build` matches both `./build/` and `./app/build/`.